src/views/Statistics.vue

```diff
@@ -2,8 +2,15 @@
      Use of this source code is governed by a BSD-style license that can be
      found in the LICENSE file. -->
 
+// TODO: Style this, and add tabs for User & Team.
```

Comment:
this is the wrong style of comment for this portion of the file -- use `<!-- TODO: ... -->`

Reply:
Changed comment style.